src/receipt/receipt_std.rs

```diff
@@ -422,11 +422,9 @@ impl io::Write for Receipt {
 impl bytes::Deserializable for Receipt {
     fn from_bytes(bytes: &[u8]) -> io::Result<Self> {
         let mut instance = Self::ret(Default::default(), 0, 0, 0);
-
```

Comment:
[nit] these changes seem unrelated to the PR?

Reply:
Nah!